content/tracing/search.md

```diff
@@ -33,9 +33,11 @@ further_reading:
 
 Use Trace Search & Analytics to filter application performance metrics and [APM events][8] by user-defined tags. It allows deep exploration of the web requests flowing through your service.
 
-Trace Search & Analytics can be enabled per APM service and per host. A service on which it is enabled exposes all its APM Events to Datadog.
+Trace Search & Analytics can be enabled per APM service and per host. A service on which it is enabled exposes all its APM Events to Datadog. 
 
-In this view you can:
+Downstream services like databases and cache layers won't be in the list of available services (as they don't generate traces on their own), but their information will be picked up by the top level services that call them.
```

Comment:
r/`but their information will be picked up by the top level services that call them.`/but their information is picked up by the top level services that call them.`